support-frontend/assets/pages/digital-subscription-checkout/digitalSubscriptionCheckout.jsx

```diff
@@ -72,7 +72,7 @@ const content = (
     >
       <CheckoutStage
         checkoutForm={<CheckoutForm />}
-        thankYouContentPending={<ThankYouPendingContent {...thankyouProps} />}
+        thankYouContentPending={<ThankYouPendingContent includePaymentCopy={true} {...thankyouProps} />}
         thankYouContent={<ThankYouContent {...thankyouProps} />}
```

Comment:
Do we want to pass this includePaymentCopy prop to < ThankYouContent /> too? If so you could add includePaymentCopy: false to the thankyouProps object and save having to put an inline prop here.

Reply:
it would be `true` rather than false in this case, but the answer is the same as I mentioned in the other case.

Reply:
Based on comments elsewhere this is fine as is.